### ISSUE TITLE
Year filter in facets list

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH/AAPB2#408]
Remove years from list after 2015.

**Comment by mccalluc** [comment]
Huh. `cleaner.rb` should have removed dates coming from the future:
```ruby
     match_no_report(doc, '/pbcoreAssetDate') { |node|
      match = node.text.match(/^(\d{4})/)
      Cleaner.delete(node) unless match && match[1].to_i.between?(1900, Time.now.year)
    }
```

I reingested locally, and the date was removed, as expected: `ruby scripts/download_clean_ingest.rb --same-mount --skip-sitemap --ids cpb-aacip_43-225b079t`

I think that check has been in there since ingesting began, so I'm a little confused. I'm going to log in and reingest just those right now and see if it makes a different. Stay tuned...
